include/drivers/gpio.h

```diff
@@ -55,12 +55,19 @@ extern "C" {
 /* Initializes output to a high state. */
 #define GPIO_OUTPUT_INIT_HIGH   (1U << 11)
 
+/* Initializes output based on logic level */
+#define GPIO_OUTPUT_INIT_LOGICAL (1U << 23)
+
 /** @endcond */
 
 /** Configures GPIO pin as output and initializes it to a low state. */
-#define GPIO_OUTPUT_LOW         (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW)
+#define GPIO_OUTPUT_LOW      (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW)
 /** Configures GPIO pin as output and initializes it to a high state. */
-#define GPIO_OUTPUT_HIGH        (GPIO_OUTPUT | GPIO_OUTPUT_INIT_HIGH)
+#define GPIO_OUTPUT_HIGH     (GPIO_OUTPUT | GPIO_OUTPUT_INIT_HIGH)
+/** Configures GPIO pin as output and initializes it to a logic 0. */
+#define GPIO_OUTPUT_INACTIVE (GPIO_OUTPUT_INIT_LOGICAL | GPIO_OUTPUT_LOW)
```

Comment:
Other user facing defines that have doxygen documentation, like `GPIO_INT_EDGE_TO_ACTIVE` always specify _all_ the bit flags they depend on. I propose to keep the convention and write
```
#define GPIO_OUTPUT_INACTIVE    (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOGICAL | \
				 GPIO_OUTPUT_INIT_LOW)
```
Otherwise we are defining compound flags in terms of other compound flags.